src/translations/fr.json

```diff
@@ -280,6 +280,8 @@
   "vlId": "ID poste",
   "p0": "P constant",
   "q0": "Q constant",
+  "batteryTargetP": "Puissance active constante",
+  "batteryTargetQ": "Puissance réactive constante",
```

Comment:
US are updated recently by Chistine, we do not need these keys, can reuse targetP, targetQ

In addition, we need another key, ie. 'filterNominalVoltage: Tension nominale poste' which is different to the existing key nominalVoltage

Reply:
Done.